### ISSUE TITLE
Issue with tile images on zoom/renew

### ISSUE THREAD

**Issue by dbf256** [openstreetmap/operations#503]
Hi,
I start to see couple of issues with tile images at openstreetmap.org recently

1. Tile images are inconsistent, looks like some of them are more up-to date, so there are some artifacts on borders, example attached.
![tiles](https://user-images.githubusercontent.com/2550295/105486684-f4c51e00-5cbf-11eb-8516-4d0e23620e58.jpg)
2. Earlier I was able to see updated tiles just after I upload changes from JOSM (usually one Shift+F5 was required to refresh page), but for now it doesn't help - changes appears with delays and for different zooms.

Was something changed that could affect quality of tile distribution?

I can do tests/provide additional details if required. Thanks in advance.

**Comment by tomhughes** [comment]
There is absolutely no point posting a random image without even giving the vaguest clue what part of the map you are looking at, what zoom level, or any other information that might debug it.

In general rerendering is not instance and it make take a few reloads before all tiles in a given area have rerendered because there are multiple render servers and multiple caches.

I'm guessing you are most likely in Europe, as that is where you are most likely to see this sort of behaviour as there are four render servers and requests are randomly distributed over them and there may be differences in how behind they (though measured in seconds, up to a few minutes) and how busy they are and therefore how likely they are to be able to do an immediate render of a dirty tile.

Basically be patient - in most cases changes will be rendered within a few minutes and it will almost always happen within a few hours.

**Comment by dbf256** [comment]
@tomhughes, thanks for quick reply. I used this picture as an illustration for the statement "Tile images are inconsistent, looks like some of them are more up-to date, so there are some artifacts on borders" to explain what I mean better.

Here is a point on map: https://www.openstreetmap.org/#map=18/44.42252/38.20528
If you change zoom around this value you can see some breaks on borders of tiles.
Also I noticed that when I just reload page images are changing and on some place not consistent, a series of screenshots is here:

https://clip2net.com/s/4aERyuX
https://clip2net.com/s/4aERBJq
https://clip2net.com/s/4aERCM0

(just in case it is not reproduced on your side but is helpful)

Thanks.

**Comment by tomhughes** [comment]
A simple shift reload on that page to flush the caches fixed all the issues for me...
